lib/puppet/indirector/file_bucket_file/file.rb

```diff
@@ -158,7 +158,7 @@ def save_to_disk(bucket_file, files_original_path, contents_file, paths_file)
         Puppet::FileSystem.exclusive_open(paths_file, 0640, 'a+:external') do |f|
           if Puppet::FileSystem.exist?(contents_file)
             if verify_identical_file(contents_file, bucket_file)
-              Puppet.info "FileBucket got a duplicate file #{bucket_file.checksum}"
+              Puppet.info _("FileBucket got a duplicate file %{file_checksum}") % { file_checksum: bucket_file.checksum }
```

Comment:
Is `FileBucket` the name of a method here? Or is this something that should be translated?

Reply:
Yeah, I thought I added a comment for that. Let me fix it.